test-case/check-signal-stop-start.sh

```diff
@@ -61,15 +63,14 @@ setup_kernel_check_point
 func_stop_start_pipeline()
 {
     local i=1
-    while ( [ $i -le $count ] && [ "$(ps -p $pid --no-header)" ] )
+    while ( [ $i -le $count ] )
     do
         # check aplay/arecord process state
-        sof-process-state.sh $cmd >/dev/null
-        if [ $? -ne 0 ]; then
-            "$cmd process is in an abnormal status"
+        sof-process-state.sh $cmd >/dev/null || {
+            dloge "$cmd process is in an abnormal status"
```

Comment:
Waiting for a process after sending `kill -9` is stupid, please drop the `wait`. 

(the `2>/dev/null` was also stupid).

Reply:
> Waiting for a process after sending kill -9 is stupid, please drop the wait.

There's another one  at the bottom.

Reply:
thanks for pointing out